src/console-errors/3ds/support-codes/codes/002-nasc.js

```diff
@@ -2,18 +2,25 @@ const SupportCode = require('../../../common/support-code');
 
 module.exports = {
 	'0100': new SupportCode('Missing name', 'Missing description', 'The online service is unavailable.\nPlease try again later.\n\nFor help, visit support.nintendo.com', 'Missing fix', 'Missing link'),
-	'0101': new SupportCode('Missing name', 'Missing description', 'Either the network is experiencing high\ntraffic volumes or the service is down.\nPlease try again later.\n\nFor help, visit support.nintendo.com', 'Missing fix', 'Missing link'),
+	'0101': new SupportCode('Missing name', 'Game server is under maintenance', 'Either the network is experiencing high\ntraffic volumes or the service is down.\nPlease try again later.\n\nFor help, visit support.nintendo.com', 'Missing fix', 'Missing link'),
```

Comment:
Where did these descriptions come from? It looks like 0111 and 0101 were swapped here?

Reply:
Yeah the correct maintenance code is 0101, I imported this from the [Wii version of the code](https://wiimmfi.de/error?e=20101&m=json)

Reply:
Oh that's interesting! I didn't know they had a page like this. Checking it out I found that they link to [this forum post](https://forum.wii-homebrew.com/index.php/Thread/51738-Collecting-Error-Codes/#post590284) listing all error codes, I wonder how many are useful here?